odl/discr/discr_mappings.py

```diff
@@ -15,19 +15,22 @@
 # You should have received a copy of the GNU General Public License
 # along with ODL.  If not, see <http://www.gnu.org/licenses/>.
 
-# pylint: disable=abstract-method
+"""Mappings between abstract (continuous) and discrete sets.
+
+Includes grid evaluation (collocation) and various interpolation
+operators.
+"""
 
 # Imports for common Python 2/3 codebase
 from __future__ import print_function, division, absolute_import
 
 from future import standard_library
 standard_library.install_aliases()
-from builtins import str, zip
+from builtins import str, super, zip
```

Comment:
These are used only in the private classes, once per interpolator creation, i.e. once per operator evaluation.